components/hab/src/command/origin/key/list.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs;
+use std::path::Path;
+use error::Result;
+
+pub fn start(cache: &Path) -> Result<()> {
+    for entry in fs::read_dir(cache)? {
+        let entry = entry?;
+        let path = entry.path();
+
+        if path.is_file() {
+            println!("{:?}", path.file_name());
```

Comment:
I think you wanted to use `{}` instead `{:?}`, because the latter prints the debugging details, no? Anyway, I wonder if you shouldn't use `common::ui::UI` for printing stuff on the terminal.